app/src/main/res/menu/room_menu.xml

```diff
@@ -19,14 +19,6 @@
           android:visible="false"
           app:showAsAction="ifRoom"/>
 
-    <item android:id="@+id/pause_audio_menu_item"
```

Comment:
Why are these removed? They are handy for QE folks.

Reply:
Ah yeah, I thought we agreed a while back that it is okay to remove these? Just went ahead and took care of it as part of this PR. Happy to add it back in if needed if QE uses these menu items often. I think these options are confusing to the the average user compared to the FABs that turn off/on video and audio.

Reply:
they can be useful for debugging differences between unpublishing versus muting/unmuting. Let's add them back. They are not for a typical user, which is why they are in the expanded menu.